plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/TestingPinotCluster.java

```diff
@@ -64,8 +66,9 @@
     private final GenericContainer<?> zookeeper;
     private final HttpClient httpClient;
     private final Closer closer = Closer.create();
+    private final boolean securedController;
 
-    public TestingPinotCluster(Network network)
+    public TestingPinotCluster(Network network, boolean securedController, boolean securedBroker)
```

Comment:
No need to split into two boolean variables since they are always same.

Reply:
I won't have to write test for combinations of secured and non-secured then?

Reply:
Such test isn't required.